### DISCUSSION TITLE
Notes panel suggestions

### DISCUSSION THREAD

**Discussion by jack-sprat** [freeplane/freeplane#271] [Ideas]
**TL;DR: collapsible note panel; menu and toolbar buttons in notes panel**

The collapsible LH and RH icons and tools panels are familiar from, e.g. LibreOffice, which I like. However, the Note editor panel must be activated from the View > Notes menu (or CTRL+>) - it would be great it if also worked as a collapsible panel.

Generally I'm just using notes for unformatted text (or use Markup), so have the menu and toolbar off; again, to activate these requires View > Notes menu - it would be great if they could be activated with buttons within the notes panel.

(This is a great project - as a long-time Freemind user, it's an easy upgrade - thanks)

**Comment by EdoFro** [comment]
I like both ideas

I would also like that the user can undock the note's panel as an independent window, so you can see the FP UI with the map in one monitor and the note's panel in another

**Reply by kenfhill84083** [reply]
Have you tried : **Edit > Note > Edit note in dialog**?  It displays the node's note in a floating dialog that can be dragged to another monitor or moved there using shift+windows+arrow (left/right).
